docker/lr-pb/add_pac_bio_rg_to_bam.py

```diff
@@ -0,0 +1,168 @@
+import pysam
```

Comment:
I'm going to assume you did basic testing on these.
I do have one question: why not use `samtools reheader` to add/change a headerline? This python script will ultimately need testing, whereas with `samtools`, one just needs to test the WDL task.

Reply:
Correct - I tested this on a few bam files.

I'm not sure why - at the time this seemed like the lowest barrier of entry to getting this done.

I added issue #348 to track this.